src/main/java/io/vertx/mqtt/impl/MqttServerConnection.java

```diff
@@ -179,7 +179,7 @@ synchronized void handleMessage(Object msg) {
 
     } else {
 
-      this.chctx.fireExceptionCaught(new Exception("Wrong message type"));
+      this.chctx.fireExceptionCaught(new Exception("Wrong message type " + msg.getClass().getName()));
```

Comment:
Yes I think that it could be useful here.
I also noticed that in the `default` case above there is the same message which is not useful because we know that the class is an `MqttMessage` and we should print the message type using `mqttMessage.fixedHeader().messageType()`. Can you make this change and then squashing this two commits ? I'll be happy to merge :-)

Reply:
Done